hydroflow/src/lang/monotonic_map.rs

```diff
@@ -0,0 +1,50 @@
+use super::clear::Clear;
+
```

Comment:
nit: doesn't have to be in this PR but a comment or example of where this thing is useful would be good